src/setup/auth-service.js

```diff
@@ -45,7 +45,7 @@ function checkAuth (auth) {
     if (!util.isObject(auth)) {
         throw new CartoValidationError(`${cvt.INCORRECT_TYPE} 'auth' property must be an object.`);
     }
-    auth.username = auth.user; // API adapter
+    auth.username = util.isDefined(auth.username) ? auth.username : auth.user; // backwards compatibility
```

Comment:
`util.isDefined(` could we deprecate it?

Reply:
I have just added it to utils... as I was using `!util.isUndefined` ... but I can remove it if you don't like it very much! don't you like that sugar 😄 ?

Reply:
Well... I don't like it much. I think it is more verbose than directly comparing it, and, if the language provides a reasonable to do it, I prefer that over an util

Reply:
I see your point, you prefer not using a new util. I prefer `isDefined` to `!isUndefined`, cause IMO negatives are a bit harder to understand than positives. 

But it is not a big deal, so I'm gonna remove the new method. 
I'll also flip the if, cause I always try to express conditionals as positives.

Thx for your review!